WebApiCoreSeed.WebApi/appsettings.Development.json

```diff
@@ -1,6 +1,7 @@
 ﻿{
   "ConnectionStrings": {
-    "DefaultConnection": "Server=(localdb)\\mssqllocaldb;Database=WebApiCoreSeedDb;Trusted_Connection=True;MultipleActiveResultSets=true"
+    //"DefaultConnection": "Server=(localdb)\\mssqllocaldb;Database=WebApiCoreSeedDb;Trusted_Connection=True;MultipleActiveResultSets=true",

```

Comment:
Yes, is for my instance in my equipment i will leave the original Connection and correct json